### PR TITLE
League Gothic: Version 2.001 added

### PR DESCRIPTION
**Pull request by emmamarichal**
60da752: [gftools-packager] League Gothic: Version 2.001 added

* League Gothic Version 2.001 taken from the upstream repo https://github.com/sursly/league-gothic at commit https://github.com/sursly/league-gothic/commit/f1d4a1c8c1db3c4a32cc13763c5fbb1776b5d478.

 d3781c2: [gftools-packager] ofl/leaguegothic remove METADATA "source".  google/fonts#2587

### CODE REVIEW

**Comment by RosaWagner** [comment]
Ah I think Fontbakery is confused because there is not a real weight axis, but the usweightclass is 400. I am actually not sure what is the exact process in that case, so I am invoking @m4rc1e. But we'll see that next week.

File: ofl/leaguegothic/METADATA.pb [hunk 1]
```diff
@@ -1,27 +1,23 @@
 name: "League Gothic"
-designer: "League Of Movable Type, Mirko Velimirovic"
+designer: "Tyler Finck, Caroline Hadilaksono, Micah Rich"
 license: "OFL"
 category: "SANS_SERIF"
 date_added: "2020-01-30"
```
Comment (davelab6) [inline ofl/leaguegothic/METADATA.pb:5]: Is this correct?

Comment (davelab6) [reply ofl/leaguegothic/METADATA.pb:5]: Should be 2022-12-13 I think

Comment (RosaWagner) [reply ofl/leaguegothic/METADATA.pb:5]: no

**Comment by davelab6** [comment]
@felipesanches please resolve the fontbakery issue

**Comment by felipesanches** [comment]
> @felipesanches please resolve the fontbakery issue

I fixed it yesterday

**Comment by felipesanches** [comment]
the fontbakery fix was already merged and will be included on the next release (FB v0.8.5)

**Pull request merged**